modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/application/HeadlessBuilderApplicationFactoryImpl.java

```diff
@@ -317,6 +332,9 @@ private void _validate(PathItem pathItem) throws Exception {
 		}
 	}
 
+	@Reference
+	private InfoItemServiceRegistry _infoItemServiceRegistry;
```

Comment:
@W2jx As Shuyang said, please don't inline this util since it results in 2 references to the registry instead of 1 before your change.

Only inline when the util is only used by one component.